react-native-in-app-payments-quickstart/app/screens/HomeScreen.tsx

```diff
@@ -209,23 +181,23 @@ export default class HomeScreen extends Component {
     }
   }
 
-  onGooglePayRequestNonceFailure(errorInfo) {
+  const onGooglePayRequestNonceFailure = (errorInfo: any) => {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.